src/base.rs

```diff
@@ -118,7 +118,8 @@
     target_arch = "arm",
     target_arch = "aarch64",
     target_arch = "x86",
-    target_arch = "x86_64"
+    target_arch = "x86_64",
+    target_arch = "riscv64"
```

Comment:
Can you move this up to keep it ordered?
Any reason not to add 32 and 128?

Reply:
Thanks, re-ordered. I didn't add 32-bit support because I don't think its likely to be used (EDKII for instance only has 64-bit support.) And for 128-bit there is no Rust toolchain yet.